### PR TITLE
Feature/slash command

### PR DESCRIPTION
**Pull request by DanielBoomer**
Adding a slash command to query which team member is up next in line

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,6 +68,27 @@ app.get('/get', function (req, res) {
   res.send(getCurrentTeamMember());
 });
 
+// Handling slah commands in Slack
+app.get('/slack', function (req, res) {
+  res.sendStatus(200)
+})
+
+app.post(bodyParser.urlencoded({extended:true}), function(req, res) {
```
Comment (TheConnMan) [inline index.js:76]: Does this POST respond on `/slack`? I see the GET above is updated (and still can be removed), but I'd expect the POST to look like `app.post('/slack', bodyParser......)`.

File: index.js [hunk 2]
```diff
@@ -82,6 +108,9 @@ function getPreviousTeamMember() {
 function getCurrentTeamMember() {
   return TEAM[index % TEAM.length];
 }
+function listCurrentTeamMember() {
```
Comment (TheConnMan) [inline index.js]: Is this function used? I don't see a reference to it. If not, it can be removed.

Comment (DanielBoomer) [reply index.js]: You know what. I put it in there to use it, but never did.  It can be taken out

File: index.js (hunk 1 above)
Comment (DanielBoomer) [reply index.js:76]: It has worked in all of my tests

File: index.js (hunk 2 above)
Comment (TheConnMan) [reply index.js]: Cool, please rip it out then, no need to keep around dead code.

**Review by TheConnMan — APPROVED** [review]

**Pull request merged**